apps/spica/src/styles.scss

```diff
@@ -86,19 +98,19 @@ mat-card {
 }
 
 mat-form-field {
-  .mat-form-field-outline {
+  .mdc-form-field__outline {
     background: rgba(var(--background-app-bar-rgb), calc(0.22 + var(--tone-diff)));
     border-radius: 5px;
     top: 0.75em !important;
     > * {
       visibility: hidden;
     }
   }
-  .mat-form-field-label {
+  .mdc-form-field__label {
     top: 2.14375em !important;
   }
-  &.mat-form-field-appearance-outline.mat-form-field-can-float.mat-form-field-should-float {
-    .mat-form-field-label {
+  &.mdc-form-field--outlined.mdc-form-field--can-float.mdc-form-field--should-float {
+    .mdc-form-field__label {
```

Comment:
Could you explain or refer to why these class names include kebab and snake case naming? Also, some of them include double dashes between words, are they running as expected?

Reply:
This is the result of the migration script. 
<img width="918" alt="Screenshot 2024-12-31 at 11 06 59" src="https://github.com/user-attachments/assets/5c9bd437-5941-477c-ba53-9859fbdaf47d" />